source/tutorials/quickstart-flakes.md

```diff
@@ -0,0 +1,148 @@
+# Quickstart Guide to Flakes
+
+This guide focuses on things that you need to know as a beginner,
+directing you where to go for more information on each particular topic.
+
```

Comment:
As [suggested here](https://github.com/NixOS/nix.dev/pull/265/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R150-R152), add prerequisites to being able to make use of this guide.


```suggestion

## Prerequisites
- reading the Nix language <!-- TODO: link to currently recommended resource -->

```